applications/luci-app-vpnbypass/Makefile

```diff
@@ -8,9 +8,9 @@ PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
 
 LUCI_TITLE:=VPN Bypass Web UI
 LUCI_DESCRIPTION:=Provides Web UI for VPNBypass service.
-LUCI_DEPENDS:=+luci +vpnbypass
+LUCI_DEPENDS:=+luci-mod-admin-full +vpnbypass
 LUCI_PKGARCH:=all
-PKG_RELEASE:=4
+PKG_RELEASE:=9
```

Comment:
Why do you increase the PKG_RELEASE by 5?

Reply:
I have my own repo to which I've been releasing the small updates and now getting it in sync.

Reply:
ok